dev/package-examples/auditd-2.0.4/docs/README.md

```diff
@@ -34,51 +33,45 @@ registered to receive data from the Linux Audit Framework. You can use these
 commands to see if the `auditd` service is running and stop it:
 
 * See if `auditd` is running:
-+
-[source,shell]
------
+
+```
```

Comment:
@hbharding @neptunian Do we we have any options here to keep nice code highlighting also in markdown. Kind of indicating if it is shell, yaml, json etc?